### PR TITLE
[Preset] Disable verify dwarf for toolchain preset

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
Requires: https://github.com/apple/swift/pull/72486/

File: utils/build-presets.ini [hunk 1]
```diff
@@ -1301,6 +1301,8 @@ extra-cmake-options=
    -DCMAKE_C_FLAGS="-gline-tables-only"
    -DCMAKE_CXX_FLAGS="-gline-tables-only"
 
+dsymutil-args="--verify-dwarf=none"
```
Comment (edymtt) [inline utils/build-presets.ini:1304]: The flag has been renamed in the original PR

```suggestion
extra-dsymutil-args="--verify-dwarf=none"
```

**Comment by shahmishal** [comment]
@swift-ci build toolchain

**Comment by shahmishal** [comment]
@swift-ci smoke test

**Comment by shahmishal** [comment]
```
7.3%              	 913.93                	 macosx-arm64-extractsymbols
7.0%              	 886.72                	 macosx-x86_64-extractsymbols
```

**Pull request merged**